prestodb/client.py

```diff
@@ -38,8 +38,11 @@
 from typing import Any, Dict, List, Optional, Text, Tuple, Union  # NOQA for mypy types
 
 import prestodb.logging
-import prestodb.redirect
+import google.auth.transport.requests
 import requests
+from google.oauth2 import service_account
```

Comment:
Wow...... Directly including google oauth libs in the header of this file seems to be dangerous.

Why do we need to include them here? I believe they're not hard dependencies for the python client as some users are not deploying Presto on the GCP. What about using a conditional import when necessary?